.yamllint.yml

```diff
@@ -1,2 +1,5 @@
 ---
 extends: relaxed
+
+rules:
+  trailing-spaces: disable
```

Comment:
Line length isn't too egregious, but for hygiene purposes I think it is good to enforce no trailing spaces -- especially because of how "flexible" YAML can be regarding implicit strings. If someone wants extra spaces they can always quote the string and put their extra spaces in the quotes.

Reply:
there were several files that failed validation with this enabled. I'll take a look to see how much effort it would be to fix them

Reply:
Yeah, I think the best thing here would be to just fix the bad files.

If you open the files in VS Code you should be able to open the command palette and type `Trim trailing whitespace`
![trim trailing whitespace](https://i.stack.imgur.com/wmfl7.png)

That should be enough to fix it for the offending files.